source/_components/spotify.markdown

```diff
@@ -123,6 +123,19 @@ which are part of the
 service. You can test this from the services
 control panel in the Home Assistant frontend.
 
+## Services
+Extra services besides the default ones in component [Media Player component](/components/media_player/).
+
+### Service `play_playlist`
+
+Play a Spotify playlist, with some extra features from generic service `"media_player.media_play"`.
```

Comment:
What extra features are we referring to?

Reply:
You can start on random position of the playlist, generic service `media_player.media_play` doesn't allow that.

Reply:
I think we should be specific, and just say: ", with an option to start on a random position of the playlist.".

Reply:
Please check if is better.